sbin/build.sh

```diff
@@ -887,23 +887,30 @@ buildCyclonedxLib() {
   else
     ANTBUILDFILE="${CYCLONEDB_DIR}/build.xml"
   fi
+
+  # Do we have a local cache for the dependency jars?
+  local localJarCacheOption=""
```

Comment:
Can we use a default value for this since it's likely to be in a common location on all machines, and it would be good to use the cached version automatically if it is available instead of requiring it to use a build option?

Reply:
@sxa bearing in mind "users" can use this script, do you see it a reasonable default for users in general?

Reply:
also what should the default be, and i'd need to add platform specific differences here..?

Reply:
my assumption was we would specify the adoptium ci defaults in the ci-jenkins-pipelines job configs

Reply:
If we're not specifying a default location within the script itself, I advise that we add error checking for a blank value.

Example below.

Reply:
The defaults should be consistent with wherever we're going to install them in the playbooks.

So yes, potentially different for each platform.

The fact that users can use this is precisely why we should have a default - so the users can use the defaults on a system configured with the playbooks, such as the simplest option where they are using it container images.

Reply:
I've added some defaults, we can correct them when we get to the infra issue if they don't match.

Reply:
SGTM. I definitely wouldn't want it to be under $HOME since that will typically be as location writable by the build processes, but I'm ok with merging this for now and changing later since we have these placeholders now 😀